packaging/package.yml/en.md

```diff
@@ -44,44 +42,43 @@ install    : |
 **extract** | `bool` | Set to `no` to disable automatic source extraction.
-**autodep** | `bool` | Set to `no` to disable automatic binary dependency resolution at build time
-**emul32** | `bool` | Set to `yes` to enable an `-m32` build (32-bit libs)
-**libsplit** | `bool` | Set to `no` to disable splitting of libraries into `devel` sub-packages
+**autodep** | `bool` | Set to `no` to disable automatic binary dependency resolution at build time.
+**emul32** | `bool` | Set to `yes` to enable an `-m32` build (32-bit libs).
+**libsplit** | `bool` | Set to `no` to disable splitting of libraries into `devel` sub-packages.
+**conflicts** | `string(s)` | Specify packages that cannot be installed together with this one.
 **optimize** | `list` | Specify preset keys to modify compiler and linker flags during build. You can learn more [here](/articles/packaging/package.yml/en/#optimize-values).
 **builddeps** | `list` | Specify build dependencies for the package. You can learn more [here](/articles/packaging/packaging-practices/en/#build-dependencies).
 **rundeps** | `dict(s)` | Specify further runtime dependencies for the packages. You can learn more [here](/articles/packaging/packaging-practices/en/#runtime-dependencies).
-**replaces** | `dict(s)` | Replace one package with another, used when renaming or deprecating packages for clean upgrade paths
-**patterns** | `dict(s)` | Allows fine grained control over file placement within the package or sub-packages. Useful for packages that are development only (i.e. `/usr/bin` files)
+**replaces** | `dict(s)` | Replace one package with another, used when renaming or deprecating packages for clean upgrade paths.
+**patterns** | `dict(s)` | Allows fine grained control over file placement within the package or sub-packages. Useful for packages that are development only (i.e. `/usr/bin` files).
 **environment** | `unicode` | Specify code that will be exported to all build steps of the build (i.e. exporting variables for the entire build).
-**networking** | `bool` | Set to `yes` to enable networking within solbuild
-**homepage** | `string` | Provides a link to the package's hompage in the software center.
+**networking** | `bool` | Set to `yes` to enable networking within solbuild.
+**homepage** | `string` | Provides a link to the package's hompage in the Software Center.
 
 ### Build step keys, optional
 
-Note that each step in itself is optional, however all can be used. The value of each of these keys is merged into a build script that is executed for each stage of the build.
+The build step sections are all considered optional, however the absence of the `install` step will result in no package generated. Each of these keys contains content that will be placed within a script and executed within a controlled environment to perform the package build. To all intents and purposes, they are Bash scripts with a predefined environment.
```

Comment:
The `build` step is the build step, the rests aren't, so we should probably refine the language in this regard to making `### Packaging Step Keys`

`To all intents` should probably be fixed to `For all intents`